package.json

```diff
@@ -17,7 +21,6 @@
   "scripts": {
     "dev": "storybook dev -p 6006",
     "build": "rimraf dist && vite build",
-    "prepare": "rimraf dist && vite build",
```

Comment:
What's the reasoning behind this? This would prevent from using it directly as a git dependency